docs/explanation/metrics.md

```diff
@@ -50,14 +50,16 @@ However, there are a bunch of other tables in there as well.  In short, only exp
 
 Activity events must have a `uid` property. Some call sites also set `device_id`. Properties containing user agent information and the OAuth client id are set automatically based on the request object.
 
-Flow events have their properties set automatically too, either from the request object or, if there is an authentication token of some kind, from data stored against the uid and token id in memcached. This data is widely referred to as `metricsContext` throughout the codebase and has a lifetime of 2 hours, which is kind of an arbitrary limit that allows us to fetch data for most flows while ignoring dubious data from stale or non-authentic sources. The `metricsContext` is written to memcached via calls to `request.stashMetricsContext` (where it’s copied from the request data) and `request.propagateMetricsContext` (where it’s copied from some other stashed data). Those functions are both defined in `lib/metrics/context.js` and decorated onto the request object in `lib/server.js`.
+Flow events have their properties set automatically as well, either from the request object or, if there is an authentication token of some kind, from data stored against the uid and token id in memcached. This data is widely referred to as `metricsContext` throughout the codebase and has a lifetime of 2 hours, which is an arbitrary limit that allows us to fetch data for most flows while ignoring dubious data from stale or non-authentic sources. The `metricsContext` is written to memcached via calls to `request.stashMetricsContext` (where it’s copied from the request data) and `request.propagateMetricsContext` (where it’s copied from some other stashed data). Those functions are both defined in `lib/metrics/context.js` and decorated onto the request object in `lib/server.js`.
 
 Amplitude events are not explicitly emitted (note the exception in [fxa-payments-server](#payments-server)), because they can all be mapped to from previously-existing activity or flow events. Those mappings are defined by structures called `EVENTS` and `FUZZY_EVENTS` in `lib/metrics/amplitude.js`. Any event matching a string key from `EVENTS` or a regex key from `FUZZY_EVENTS` gets transformed into the corresponding event by common code in the `fxa-shared` package (it’s also used by the content server). Most of the properties on those events are set automatically, although there are still some cases where data must be set explicitly at the call site.
 
+All events are validated against a shared schema, then logged to `stdout`. These logs are loaded into BigQuery into partitioned tables. As BigQuery uses schema auto-detection, if the BigQuery data type changes, the log line won't go to the partitioned `stdout` table, but instead to an error table.
```

Comment:
> All events are validated against a shared schema, then logged to `stdout`.

This makes it seem like invalid events aren't logged, though I left it intentionally ambiguous, since currently we log invalid events anyway but also log the validation error. I'm thinking it might be good to mention this for auth server FxA events.

The metrics epic will make it so that P1 SubPlat events on both the auth and payments server will only be logged when valid, so we don't need to change the language for the payments server, but making that distinction here for the auth server makes more sense, since there will be FxA and SubPlat events in the future. Currently prior to this metrics epic, there have only been FxA events on the auth server.